server/routes.js

```diff
@@ -422,6 +425,64 @@ exports.isomorphicRoutes = function isomorphicRoutes(
       )
     );
   });
+  function prepareSlug(urls, req) {
+    const destsClone = urls;
+    urls.forEach((item) => {
+      if (req.params[item]) {
+        destsClone.splice(destsClone.indexOf(item), 1, `${req.params[item]}`);
+      }
+    });
+    return `${destsClone.join("/")}`;
+  }
+  function getUrlRedirect(sourceUrlArray, chunkUrls) {
+    app.get(sourceUrlArray, (req, res, next) => {
+      const query = url.parse(req.url, true) || {};
+      const search = query.search || "";
+      if (req.params) {
+        chunkUrls.forEach((chunkUrl) => {
+          const destS = chunkUrl.destinationUrl.split("/:");
```

Comment:
Make sure the error handling is done properly. What happens when destinationUrl is not there? Log the error to the user so that they can fix it.

Reply:
Also check the naming. destS is something no one will understand.